index.php

```diff
@@ -12,7 +17,32 @@
             <h1> Flock Information Database Home </h1>
         </div>        
         <div class="content">
-            <h2> Stick a login form here </h2>
+            
+            <center>
+                <h3>User Login</h3>
+                <hr />
+                <form method="post" action="redirectPostLogin.php">
+                    <table border="0">
```

Comment:
The `border="0"` belongs in the stylesheet. You should give this table an id though
`<table id="login-table">`
Then in the stylesheet under the index.php comment
```css
#login-table {
   border: 0px;
}
```